install/bootstrap_livecd_remote_iso.md

```diff
@@ -562,14 +562,7 @@ Finally, cleanup the shim:
    pit# rpm -Uvh --force $(find ${CSM_PATH}/rpm/ -name "goss-servers*.rpm" | sort -V | tail -1)
    pit# rpm -Uvh --force $(find ${CSM_PATH}/rpm/ -name "csm-testing*.rpm" | sort -V | tail -1)   
    ```
-
-1. Verify the system:
-
-   ```bash
-   pit# csi pit validate --network
-   pit# csi pit validate --services
-   ```
-
+   
 1. Follow directions in the output from the 'csi pit validate' commands for failed validations before continuing.
```

Comment:
This line should be removed as well since there will no longer be 'csi pit validate' commands reference before it.

Reply:
Good catch.  Fixed.